vm/systemSmartContracts/delegationManager_test.go

```diff
@@ -562,15 +583,51 @@ func TestDelegationManagerSystemSC_ExecuteChangeMinDeposit(t *testing.T) {
 	args.Eei = eei
 
 	dm, _ := NewDelegationManagerSystemSC(args)
-	vmInput := getDefaultVmInputForDelegationManager("changeMinDeposit", [][]byte{{25}})
-	vmInput.CallerAddr = dm.delegationMgrSCAddress
-	_ = dm.saveDelegationManagementData(&DelegationManagement{MinDeposit: big.NewInt(0)})
+	vmInput := getDefaultVmInputForDelegationManager("changeMinDelegationAmount", [][]byte{})
+	vmInput.Arguments = [][]byte{{25}}
+	vmInput.CallerAddr = configChangeAddress
 
+	_ = dm.saveDelegationManagementData(&DelegationManagement{MinDelegationAmount: big.NewInt(25)})
+	dm.marshalizer = &mock.MarshalizerStub{
+		MarshalCalled: func(obj interface{}) ([]byte, error) {
```

Comment:
I can't believe you tested marshalizer error ==))

Reply:
The reason I've tested this is that:
1. I proved that the code is bug free there :joy: 
2. Code cov would leave me alone